File: 3824-number-of-unique-xor-triplets-i/number-of-unique-xor-triplets-i.java
class Solution {
    public int uniqueXorTriplets(int[] nums) {
        int i, j, ans = 0;
        if (nums.length <= 2)
            return nums.length;
        for (i = 0; i < nums.length; i++) {
            ans |= nums[i];
        }
        return ans + 1;
    }
}

File: 3824-number-of-unique-xor-triplets-i/number-of-unique-xor-triplets-i.py
class Solution(object):
    def uniqueXorTriplets(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n=len(nums)
        if n<=2:
            return n
        ans=nums[0]
        for i in range(1,n):
            ans|=nums[i]    
        return ans+1    
        